taxonomies/categories.txt

```diff
@@ -35826,12 +36363,14 @@ zh:鲜肉
 
 <en:Fresh meats
 en:Fresh ground steaks
+de:Frische Hacksteaks
 fi:Tuoreet jauhelihapihvit
 fr:Steaks hachés frais
 nl:Verse tartaarsteaks
 
 <en:Fresh meats
 en:Fresh meat preparations
+de:Frischfleischprodukte, Frischfleisch
```

Comment:
`ERROR - de:frischfleisch already is a synonym of frischfleisch (en:fresh-meats) - cannot make a synonym of frischfleischprodukte for en:fresh-meat-preparations`

Reply:
Thanks